pages/requests/new/[ware].js

```diff
@@ -111,7 +111,9 @@ const NewRequest = () => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [requestSucceeded, requestErred, requestID])
 
-  // TODO(alishaevn): update the return value after https://github.com/scientist-softserv/webstore-component-library/issues/136 is completed
+  // TODO(alishaevn): use react bs placeholder component
+  if (isLoadingInitialRequest || !wareID) return <Loading wrapperClass='item- mt-5' />
```

Comment:
this class name of `item-` doesn't seem right, just double checking with you that was intentional?

Reply:
good catch! I had to go back a few commits to figure out what this is actually supposed to be. it was removed in [this commit](https://github.com/scientist-softserv/webstore/commit/fd56f5d085d4a3671349278f7d7ed82f8d526a7a) last week.

Reply:
fixed!